src/pages/Competition/Person/index.tsx

```diff
@@ -17,6 +19,17 @@ export const byDate = (
   return aDate - bDate;
 };
 
+
+const RoundedBg = tw.span`
+  px-[6px]
+  py-[4px]
+  rounded-md
+`;
+
+const RoomColored = styled(RoundedBg)`
```

Comment:
Do you have a solution to address if the black text color doesn't work well with the chosen room color (I.E. a black room?)

Reply:
I believe it's already addressed by adding the 70% opacity to the original color code; here is how it looks like if the room's color is black:
![opacity](https://user-images.githubusercontent.com/1007485/234062529-f86d872a-e868-43c7-bf43-af37b236931e.png)

Definitely not the best color choice, but I find it readable (maybe turning down the opacity would help more?).

Reply:
That does look not too bad on the off chance that someone decides to use a dark colored room color. I'm good to merge this PR. THANKS!

Reply:
Thanks a lot for the fast review and merge Cailyn!